pkg/util/log/crash_reporting.go

```diff
@@ -97,11 +97,9 @@ var crashReports = settings.RegisterBoolSetting(
 )
 
 var crashReportURL = func() string {
-	var defaultURL string
+	defaultURL := ""
```

Comment:
I have zero preference on what form to use here, and will go with any suggested approach.